Directory.Build.props

```diff
@@ -1,13 +1,19 @@
 <Project>
-  <!-- See https://aka.ms/dotnet/msbuild/customize for more details on customizing your build -->
+
   <PropertyGroup>
     <CurrentPreviewTfm>net8.0</CurrentPreviewTfm>
     <!--<IncludePreview>true</IncludePreview>-->
     <IncludePreview Condition="'$(IncludePreview)' == ''">false</IncludePreview>
+    <ContinuousIntegrationBuild Condition="'$(CI)' == 'true'">true</ContinuousIntegrationBuild>
```

Comment:
When is $(CI) set to true?

Reply:
In particular on GitHub Actions, but in general it should be set on most CI providers (by convention).

![image](https://github.com/passwordless/passwordless-dotnet/assets/1935960/a1549fa7-e39f-4cb1-b1d6-e86247d310a1)